src/app/(public)/admin/register-animal/page.spec.tsx

```diff
@@ -0,0 +1,114 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { describe } from "node:test";
+import { beforeEach, expect, it, vi } from "vitest";
+import RegisterAnimalPage from "./page";
+import userEvent from "@testing-library/user-event";
+
+describe("pages / register animal", () => {
+  beforeEach(() => {
+    cleanup();
+  });
+
+  it("should render page with main content", () => {
+    render(<RegisterAnimalPage />);
+
+    const heading = screen.getByRole<HTMLHeadingElement>("heading", {
+      level: 1,
+      name: "Opata",
+    });
+    expect(heading).toBeInTheDocument();
+
+    const heading2 = screen.getByRole<HTMLHeadingElement>("heading", {
+      level: 2,
+      name: "Cadastrar novo animal",
+    });
+    expect(heading2).toBeInTheDocument();
+
+    const name = screen.getByLabelText<HTMLInputElement>("Nome*");
+    expect(name).toBeInTheDocument();
+    expect(name).toHaveValue("");
+
+    const age = screen.getByLabelText<HTMLInputElement>("Idade");
+    expect(age).toBeInTheDocument();
+    expect(age).toHaveValue(null);
```

Comment:
Shouldn't it be an empty string instead?

Maybe having the input value as `nullish` can cause a console error about uncontrolled vs controlled state.